### PR TITLE
Updated all models to modernize constructor initialization of propert…

### PR DESCRIPTION
**Pull request by vasont**
…ies; specifically lists to prevent errors in refactored and existing .net core Inspire code.

### CODE REVIEW

File: src/Common/ErrorResponseModel.cs [hunk 1]
```diff
@@ -12,19 +12,10 @@ namespace Vasont.Inspire.Models.Common
     /// </summary>
     public class ErrorResponseModel
     {
-        /// <summary>
-        /// Initializes a new instance of the <see cref="ErrorResponseModel"/> class.
-        /// </summary>
-        public ErrorResponseModel()
-        {
-            this.Messages = new List<ErrorModel>();
-            this.HasUnhandledException = false;
```
Comment (zeroadam) [inline src/Common/ErrorResponseModel.cs]: False is default below then?

File: src/Workflow/WorkflowActorModel.cs [hunk 2]
```diff
@@ -15,20 +15,11 @@ namespace Vasont.Inspire.Models.Workflow
...
-            this.WorkflowActorId = "_" + Guid.NewGuid().ToString();
-            this.Members = new List<WorkflowMemberModel>();
-        }
-        
         /// <summary>
         /// Gets or sets the actor unique identity value.
         /// </summary>
         [XmlAttribute(AttributeName = "id")]
-        public string WorkflowActorId { get; set; }
+        public string WorkflowActorId { get; set; } = "_" + Guid.NewGuid();
```
Comment (zeroadam) [inline src/Workflow/WorkflowActorModel.cs]: .ToString() not needed here now?

Comment (vasont) [reply src/Workflow/WorkflowActorModel.cs]: ToString is not needed.

File: src/Common/ErrorResponseModel.cs (hunk 1 above)
Comment (vasont) [reply src/Common/ErrorResponseModel.cs]: False is always default for a bool.

Comment (vasont) [reply src/Common/ErrorResponseModel.cs]: Just like 0 is always default for an int. I took that out in another place as well.

**Pull request merged**